pokemongo_bot/__init__.py

```diff
@@ -44,9 +44,9 @@ def start(self):
         random.seed()
 
     def take_step(self):
-        self.process_cells(work_on_forts=True)
```

Comment:
work_on_forts was no longer set to False anywhere, since some of the changes yesterday.